app/services/host.py

```diff
@@ -90,9 +101,10 @@ def update(self, service, ip_address, service_repo_name, port, revision, last_ch
         return True
 
     def set_tag(self, service, ip_address, tag_name, tag_value):
-        host = Host.get(service, ip_address)
-        host.tags[tag_name] = tag_value
-        host.save()
+        # TODO note that we never sweep when we do a get... is that an error?
```

Comment:
yup, sounds good to me. and as below, will go through the call sites to ensure nothing gets weird